packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -19,6 +19,7 @@ import {
     ISignalMessage,
     ITokenClaims,
 } from "@fluidframework/protocol-definitions";
+import io from "socket.io-client";
```

Comment:
I suspect this import may need to change in order for the odsp lazy-import to remain lazy (since `OdspDocumentDeltaConnection` inherits).

Reply:
Yes, that will include socket.io into bundle (that includes factory) and thus increase boot cost.